src/components/organisms/userFeedbackForm.tsx

```diff
@@ -0,0 +1,176 @@
+			Sentry.captureException(error);
+			addToast({
+				message: tErrors("errorSendingFeedback"),
+				type: "error",
+			});
+			LoggerService.error(namespaces.feedbackForm, tErrors("errorSendingFeedbackExtended", { error }));
+		} finally {
+			setIsSendingFeedback(false);
+		}
+	};
+
+	useEffect(() => {
+		if (!isFeedbackSubmitted) return;
+		const interval = setInterval(() => {
+			setTimerCount((count) => count - 1);
+		}, 1000);
+
+		return () => clearInterval(interval);
+	}, [isFeedbackSubmitted]);
+
+	useEffect(() => {
+		if (timerCount === 0) onClose();
+	}, [timerCount, onClose]);
+
+	useEffect(() => {
+		if (!isOpen) {
+			setIsFeedbackSubmitted(false);
+			setTimerCount(10);
+			reset();
+		}
+	}, [isOpen, reset]);
+
+	return (
+		<AnimatePresence>
+			{isOpen ? (
+				<motion.div
+					animate={{ x: 0 }}
+					className={cn(
+						"h-500 w-96 rounded-t-3xl border border-gray-750 bg-gray-1100 p-6 z-[500]",
+						className
+					)}
+					exit={{ x: -500 }}
+					initial={{ x: -500 }}
+					transition={{ type: "spring", stiffness: 100, damping: 15 }}
+				>
+					<div className="flex items-center justify-between gap-1">
+						<Typography className="font-averta font-bold" size="2xl">
+							{t("title")}
+						</Typography>
+						<IconButton
+							className="group ml-auto h-default-icon w-default-icon bg-gray-1250 p-0"
+							onClick={onClose}
+						>
+							<Close className="size-3 fill-gray-750 transition group-hover:fill-white" />
+						</IconButton>
+					</div>
+					<form className="mt-5 flex h-350 flex-col justify-between" onSubmit={handleSubmit(onSubmit)}>
+						{!isFeedbackSubmitted ? (
+							<div>
+								<Input
+									label={t("form.name")}
+									placeholder={t("form.placeholder.name")}
+									{...register("name")}
+									isError={!!errors.name}
+									isRequired
+								/>
+								{errors.name ? <ErrorMessage>{errors.name.message}</ErrorMessage> : null}
+								<Input
+									className="mt-6"
+									isRequired
+									label={t("form.email")}
+									placeholder={t("form.placeholder.email")}
+									{...register("email")}
+									isError={!!errors.email}
+								/>
+								{errors.email ? <ErrorMessage>{errors.email.message}</ErrorMessage> : null}
+								<Textarea
+									rows={5}
+									{...register("message")}
+									className="mt-6"
+									isError={!!errors.message}
+									isRequired
+									label={t("form.message")}
+									placeholder={t("form.placeholder.message")}
+								/>
+								{errors.message ? <ErrorMessage>{errors.message.message}</ErrorMessage> : null}
+							</div>
+						) : (
+							<div className="my-auto text-center">
+								<Typography className="font-averta font-bold leading-7" size="1.5xl">
+									{t("titleThanks")}
+								</Typography>
+								<Typography className="mt-1 font-averta font-light" size="xl">
+									{t("ourCats")}
+								</Typography>
+								<CatLookAtFishImage className="m-auto mt-4" />
+							</div>
```

Comment:
We need to remove it, let's talk about it.

Basically, the new vision is to keep the form the way it is displayed, hide the send button when the form is submitted, display the "Thank you" message and close the form after 3-4 seconds.